### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by shivangisharma01**
Cart 
image
image

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -2,7 +2,7 @@
 @import url('/Page/Login/login.css');
 @import url('/Page/Logout/logout.css');
 @import url('/Page/Signup/signup.css');
-@import url("https://robust-ui.netlify.app/css/styles.css")
+@import url("https://robust-ui.netlify.app/css/styles.css");
 @import url("https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,300;0,400;0,600;0,700;0,900;1,200&display=swap");
```
Comment (anushasunil) [inline css/style.css:6]: just a suggestion that you can import the font in either typography.css or utility.css

File: Page/Cart/cart.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+                            </span>
+                        </div>
+
+                         <button class="btn btn-primary"><i class="fa-solid fa-cart-plus"></i> ADD TO
+                            CART</button>
+                        <button class="btn btn-error"><i class="fa-solid fa-xmark"></i>
+                            Remove</button>
+                      
+                    </div>
+
```
Comment (anushasunil) [inline Page/Cart/cart.html:74]: can remove the extra spaces in between the code

File: Page/Cart/cart.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/style.css">
+    <link rel="stylesheet" href="cart.css">
+    <script src="js/fontawesomepro.js"></script>
+    <link rel="stylesheet" href="https://site-assets.fontawesome.com/releases/v6.0.0/css/all.css">
+    <title>Scentcy</title>
+    <link rel="stylesheet" href="https://robust-ui.netlify.app/components/Card/card.css" />
+    <link rel="stylesheet" href="https://robust-ui.netlify.app/components/Button/button.css" />
```
Comment (anushasunil) [inline Page/Cart/cart.html:14]: can import the component library `.css` files in a utility

**Review by anushasunil — APPROVED** [review]
Just few suggestion, rest everything looks good.

File: css/style.css (hunk 1 above)
Comment (shivangisharma01) [reply css/style.css:6]: ok

File: Page/Cart/cart.html (hunk 2 above)
Comment (shivangisharma01) [reply Page/Cart/cart.html:74]: done

**Review by shashidwivedi — APPROVED** [review]
Everything looks good, can be merged. Good use of naming convention while naming classes

**Pull request merged**